integration_tests/tests/admin/searchPlacementRequests.cy.ts

```diff
@@ -23,7 +24,11 @@ context('Search placement Requests', () => {
     cy.signIn()
 
     cy.task('stubPlacementRequestsSearch', { placementRequests })
-    cy.task('stubPlacementRequestsSearch', { placementRequests: searchResults, ...searchQuery })
+    cy.task('stubPlacementRequestsSearch', {
+      placementRequests: searchResults,
+      crnOrName: normaliseCrn(searchQuery.crnOrName),
```

Comment:
I think we want to swap lines 29 and 30 otherwise the non-normalized `crnOrName` will overwrite the normalized `crnOrName`

Reply:
Good call!